File: Questão13.py
class Matematica:
    """Classe com métodos estáticos para operações matemáticas."""

    @staticmethod
    def fatorial(n: int) -> int:
        if n == 0 {
            return 1
        }
        resultado = 1
        for i in range(1, n + 1):
            resultado *= i
        return resultado


numero = 5
print(f"O fatorial de {numero} é {Matematica.fatorial(numero)}")

File: Questão13.java
class Matematica {
    public static int fatorial(int n) {
        if (n == 0) {
            return 1;
        }
        int resultado = 1;
        for (int i = 1; i <= n; i++) {
            resultado *= i;
        }
        return resultado;
    }
}

public class Main {
    public static void main(String[] args) {
        int numero = 5;
        System.out.printf("O fatorial de %d é %d%n", numero, Matematica.fatorial(numero));
    }
}
